MediaBrowser.Model/Configuration/ServerConfiguration.cs

```diff
@@ -449,5 +449,11 @@ public string BaseUrl
         /// Gets or sets the how many metadata refreshes can run concurrently.
         /// </summary>
         public int LibraryMetadataRefreshConcurrency { get; set; }
+        
+        /// <summary>
+        /// Gets or sets the custom Chomecast appid.
+        /// </summary>
+        /// <value>The custom Chomecast appid.</value>
+        public string CustomChomecastAppid { get; set; } = string.Empty;
```

Comment:
maybe have it nullable?

Reply:
what would be the difference?

Reply:
For a field that may not be set to some valid value, it's normal to have it nullable to indicate that a value may not necessarily be set.. Just the standard way to do things..